packages/frontend/src/components/NavBar/HelpMenu/index.tsx

```diff
@@ -26,42 +26,55 @@ const HelpMenu: FC = () => {
     const { track } = useTracking();
     const { user } = useApp();
     const { projectUuid } = useParams<{ projectUuid: string }>();
+    const [hasMounted, setHasMounted] = useState<boolean>(false);
 
-    const trackNotifications = {
-        user_id: user.data?.userUuid,
-        project_id: projectUuid,
-        organization_id: user.data?.organizationUuid,
-    };
+    useEffect(() => {
+        setHasMounted(true);
```

Comment:
is this not practically the same as the other useEffect() ?, just delaying a bit more ? I don't understand why this is fixing the issue..

Reply:
@rephus  i tried that however according to TS Headway just never will exist on window

Reply:
I see, well I still got that error on `render` so this does not fix that issue completely